lib/workers/global/initialize.ts

```diff
@@ -64,6 +64,14 @@ function setGlobalHostRules(config: RenovateConfig): void {
   }
 }
 
+function configureThirdPartyLibraries(config: AllConfig): void {
+  if (!config.useCloudMetadataServices) {
```

Comment:
invert and early return to reduce nesting

Reply:
you can add a comment to clarify

Reply:
Makes sense, done!